package.json

```diff
@@ -13,7 +62,11 @@
   },
   "devDependencies": {
     "aws-sdk": "^2.590.0",
-    "eslint": "^6.7.2"
+    "eslint": "^7.4.0",
+    "eslint-config-logdna": "^2.0.0",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-sensible": "^2.2.0",
```

Comment:
should use the latest `eslint-config-logdna@4.0.2`

you should not need the other two plugins if you are using `eslint-config-logdna`, they are already dependencies.

Reply:
Done!